docs/pages/distribution/release-channels.md

```diff
@@ -102,12 +102,7 @@ Environment variables don't exist explicitly, but you can utilize release channe
 
 Say you have a workflow of releasing builds like this:
 
-- `expo publish --release-channel prod-v1`
-- `expo publish --release-channel prod-v2`
-- `expo publish --release-channel prod-v3`
-
-- `expo publish --release-channel staging-v1`
-- `expo publish --release-channel staging-v2`
+<TerminalBlock cmd={['# Publish to release channel prod-v1', 'expo publish --release-channel prod-v2', '', '# Publish to release channel prod-v2', 'expo publish --release-channel prod-v1', '', '# Publish to release channel prod-v3', 'expo publish --release-channel prod-v3', '', '', '# Publish to release channel staging-v1', 'expo publish --release-channel staging-v1', '', '# Publish to release channel staging-v2', 'expo publish --release-channel staging-v2', '']} />  
```

Comment:
```suggestion
<TerminalBlock cmd={['# Publish to release channel prod-v1', 'expo publish --release-channel prod-v2', '', '# Publish to release channel prod-v2', 'expo publish --release-channel prod-v1', '', '# Publish to release channel prod-v3', 'expo publish --release-channel prod-v3', '', '', '# Publish to release channel staging-v1', 'expo publish --release-channel staging-v1', '', '# Publish to release channel staging-v2', 'expo publish --release-channel staging-v2']} />  
```

This results in unnecessary empty line ad the end of block.

Reply:
Good catch!